docs/site/Parsing-requests.md

```diff
@@ -43,10 +43,10 @@ and raw data is parsed from request according to the specification. In the
 example above, the first parameter is from source `path`, so its value will be
 parsed from a request's path.
 
-{% include note.html title="Controller documentation" content="See [controller](Controller.md) for more details of defining an endpoint.
+{% include note.html title="Controller documentation" content=" See [controller](Controller.md) for more details of defining an endpoint.
```

Comment:
I find it weird to see a leading space after an opening `"`. How about moving the content to a new line instead?

```
{% include note.html title="Controller documentation" content="
See [controller](Controller.md) for more details of defining an endpoint.
" %}
```

I think this is the style we are already using in other places too.

The same comment applies to other changes made in this pull request.

Reply:
I searched all lb4 docs and we're using both patterns, a leading space and new lines.

Mode Controller for example:

```
{% include note.html title="Model documentation" content="See [model](Model.md) to know more details about how to decorate a model class.
" %}
```

 I'm more curious as to why that's needed but don't want to enter Jekyll/Liquid land. I can switch it all to be new lines.

Reply:
> I searched all lb4 docs and we're using both patterns, a leading space and new lines.

Fair enough 👍 

In which case, feel free to use whatever style you like more.